test/story/rewardsStory.test.ts

```diff
@@ -1,6 +1,7 @@
 /*
  *
  * @group autocompound
```

Comment:
We can delete `@group autocompound` from here and from `package.json` because it repeats `@group rewardsV2`, cannot we?

Reply:
will leave it for now, because we will need to adapt the mangata-node runners.

Reply:
ah no, your are right. removing!